src/one_dragon/base/controller/pc_screenshot/pil_screencapper.py

```
@@ -0,0 +1,48 @@
+        """
+        return True
+
+    def capture(self, rect: Rect, independent: bool = False) -> MatLike | None:
+        """使用 PIL 截图
+
+        Args:
+            rect: 截图区域
+            independent: 是否独立截图
+
+        Returns:
+            截图数组，失败返回 None
```

Ruff (0.14.10)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)